package/lib/plugin.js

```diff
@@ -23,13 +25,25 @@ export default function (ctx, inject) {
 			ctx.app.context.app.head = ctx.app.head
 		}
 
+		if(options.jsonld){
+			ctx.app.head.script.push({
+				vmid: 'ldjson-schema',
+				json: options.jsonld,
```

Comment:
Are you expecting the user to pass a json object as the as the value, or a string of the json object?

Reply:
JSON OBJECT

SAMPLE 

```
		head: function() {
			return this.$seo({
				title: 'Home Page',
				jsonld: {
						"@context":"https://schema.org",
						"@type":"Product",
						"productID":"PRODUCTID",
						"name":"NAME",
						"description":"DESCRIPTION",
						"url":"PRODUCT URL",
						"image":"IMAGE LINK",
						"brand":"BRAND",
						"offers": [
								{
								"@type": "Offer",
								"price": "PRICE",
								"priceCurrency": "INR",
								"itemCondition": "https://schema.org/NewCondition",
								"availability": "https://schema.org/InStock"
								}
							]
						}
			})
		}
```